pkg/webhook/webhookreporter.go

```diff
@@ -24,6 +25,16 @@ type WebhookReconciler struct {
 	client.Client
 }
 
+const (
+	Update string = "update"
+	Delete string = "delete"
+)
+
+type WebhookBody struct {
+	Verb   string        `json:"verb"`
+	Report client.Object `json:"report"`
```

Comment:
maybe we should change the field name to something else (`operatorObject` or else) as the `Report` already has field called  `report` and when evaluating it , it will look like `input.report.report` look strange to me , wdyt?